cmd/eksctl/main.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/weaveworks/eksctl/pkg/ctl/generate"
```

Comment:
This should go at the end. As a suggestion, can you configure your IDE to do it automatically (in Goland, you can)

Reply:
Got it, I think I didn't have my IDE correctly configured.